exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/access_store_ui.ipynb

```diff
@@ -47,25 +59,48 @@
     "    else:\n",
     "        sb_store_file_ = 'dss_config.sqlite'\n",
     "\n",
-    "    WKey = Enum('WKey', ['file', 'password', 'ok'])\n",
-    "    style = {'description_width': '140px'}\n",
-    "    inputs = {\n",
-    "        WKey.file:     widgets.Text(description='Config. File Name', value=sb_store_file_, style=style),\n",
-    "        WKey.password: widgets.Password(description='Config. Password', style=style),\n",
-    "        WKey.ok:       widgets.Button(description='OK')\n",
-    "    }\n",
+    "    ui_look = get_config_styles()\n",
+    "\n",
+    "    header_lbl = widgets.Label(value='Configuration Store', style=ui_look.header_style, layout=ui_look.header_layout)\n",
+    "    file_lbl = widgets.Label(value='File Path', style=ui_look.label_style, layout=ui_look.label_layout)\n",
+    "    password_lbl = widgets.Label(value='Password', style=ui_look.label_style, layout=ui_look.label_layout)\n",
+    "    file_txt = widgets.Text(value=sb_store_file_, style=ui_look.input_style, layout=ui_look.input_layout)\n",
+    "    password_txt = widgets.Password(style=ui_look.input_style, layout=ui_look.input_layout)\n",
+    "    open_btn = widgets.Button(description='Open', style=ui_look.button_style, layout=ui_look.button_layout)\n",
+    "\n",
+    "    # Prepare to display an error message\n",
+    "    notify_output = widgets.Output()\n",
+    "    display(notify_output)\n",
+    "\n",
+    "    @notify_output.capture()\n",
+    "    def popup_message(message):\n",
+    "        clear_output()\n",
+    "        display(Javascript(f\"alert('{message}')\"))\n",
     "\n",
     "    def open_or_create_config_store(btn):\n",
     "        global sb_config, sb_store_file\n",
-    "        sb_store_file = inputs[WKey.file].value\n",
-    "        sb_config = Secrets(Path(root_dir) / sb_store_file, inputs[WKey.password].value)\n",
-    "        # Save the file in the shared store.\n",
-    "        %store sb_store_file\n",
-    "        del sb_store_file\n",
+    "        sb_store_file = file_txt.value\n",
```

Comment:
If you prefer, then we could also add this kind of functionality as convenience to the secret store....

Reply:
I would leave it in a notebook, to be honest. It's not impossible to call the ipython implementation of "line magic", but I think you need a good reason for that.